MAINTAINERS

```diff
@@ -0,0 +1,17 @@
+# Org-Level Maintainers (in alphabetical order)
+# Pattern: [First Name] [Last Name] <[Email Address]> ([GitHub Handle])
+Feynman Zhou <feynmanzhou@microsoft.com> (@FeynmanZhou)
+Justin Cormack <justin.cormack@docker.com> (@justincormack)
+Milind Gokarn <gokarnm@amazon.com> (@gokarnm)
+Niaz Khan <niazfk@amazon.com> (@niazfk)
+Pritesh Bandi <priteshbandi@gmail.com> (@priteshbandi)
+Steve Lasker <StevenLasker@hotmail.com> (@stevelasker)
+Toddy Mladenov <toddysm@gmail.com> (@toddysm)
+Vani Rao <vaninrao@amazon.com> (@vaninrao10)
```

Comment:
Can we replace Vani with Samir as she is no longer working on this project.

Reply:
@priteshbandi the following document is our current guidance on `Maintainers`. Please take a look, and we can follow the process to nominate Samir, and move Vani to emeritus. 

https://github.com/notaryproject/.github/blob/main/GOVERNANCE.md#notary-org-maintainers

Reply:
@priteshbandi we need to have official issue filed to put @vaninrao10 in emeritus and add @iamsamirzon. If you want to submit the proposal in the `.github` repository we can start the process